jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -65,22 +90,19 @@ pipeline {
                             echo "artifactUrl (x64): ${artifactUrl}"
 
                             def integTestResults = 
-                                build job: INTEG_TEST_JOB_NAME,
+                                build job: env.INTEG_TEST_JOB_NAME,
```

Comment:
I think according to @abhinavGupta16, use `env` prefix may cause some issues with our tester. Could you help confirm it? @abhinavGupta16

Reply:
Here I wanted to ensure the right variable is used. If we go the route not worrying about the empty string case (which would require someone manually triggering the build and clearing it) then I will remove the`DEFAULT` and only use the param and remove the `env` prefix.